curriculum/challenges/english/01-responsive-web-design/registration-form/part-022.md

```diff
@@ -0,0 +1,84 @@
+---
+id: 60f85a62fb30c80bcea0cedb
+title: Part 22
+challengeType: 0
+dashedName: part-22
+---
+
+# --description--
+
+Users will be allowed to chose either a `Personal Account` or `Business Account`.
```

Comment:
I've initially read it to add these texts to the labels in this part too. No issues from tests with that, but in the next part I was confused why text was gone.

Not that big of deal, as I've misread it. One way to prevent it could be in this part just mentioning two account types, without specific names. Or it could be left alone.